docs/src/results_store.md

```diff
@@ -0,0 +1,74 @@
+# Results
+
+Reefmod Engine Outputs and scenario information are recorded from the ReefMod Engine API
+after each scenario(s) run, not before.
+
+## ResultStore Struct
```

Comment:
```suggestion
## Result Store
```

`Struct` is an implementation detail that doesn't feel right to have in user-facing documentation.

Also, "Result Store", "results store", "ResultStore", pick one and use it consistently throughout 😸